source/NVDAObjects/window/winword.py

```diff
@@ -1430,10 +1466,520 @@ def _get_WinwordWindowObject(self):
+		self.wordChartObject.SeriesCollection(self.seriesIndex).Trendlines(self.trendlineIndex).Select()
+		eventHandler.queueEvent('gainFocus',obj)
+
+	def script_previousTrendline(self, gesture):
+		if self.trendlinesCount > 1:
+			if self.trendlineIndex==1:
+				self.trendlineIndex=self.trendlinesCount
+			else:
+				self.trendlineIndex=self.trendlineIndex-1
+			self.invokeTrendline(self.trendlineIndex)
+
+	def script_nextTrendline(self, gesture):
+		if self.trendlinesCount > 1:
+			if self.trendlineIndex==self.trendlinesCount:
+				self.trendlineIndex=1
+			else:
+				self.trendlineIndex=self.trendlineIndex+1
+			self.invokeTrendline(self.trendlineIndex)
+
+	__gestures = {
+				"kb(laptop):leftArrow":"previousTrendline",
+				"kb(desktop):leftArrow":"previousTrendline",
+				"kb(laptop):rightArrow":"nextTrendline",
+				"kb(desktop):rightArrow":"nextTrendline",
+	}
+
+class WordChartElement(WordChart):
+	_axisMap={_chartConstants.xlCategory: {_chartConstants.xlPrimary: 'Primary Category Axis', _chartConstants.xlSecondary: 'Secondary Category Axis'},
+				_chartConstants.xlValue: {_chartConstants.xlPrimary: 'Primary Value Axis', _chartConstants.xlSecondary: 'Secondary Value Axis'},
+				_chartConstants.xlSeriesAxis: {_chartConstants.xlPrimary: 'Primary Series Axis', _chartConstants.xlSecondary: 'Secondary Series Axis'}
+	}
+
+	def __init__(self,windowHandle, wordApplicationObject, wordChartObject, keyIndex, elementIndex=0):
+		self.windowHandle=windowHandle
+		self.wordApplicationObject=wordApplicationObject
+		self.elementIndex=elementIndex
+		self.keyIndex=keyIndex
+		self.wordChartObject=wordChartObject
+		self.elementKeyList=[]
+		if self.wordChartObject.HasTitle:
+			self.elementKeyList.append('chartTitle')
+		# Enumerations for chart object in Excel and Word are same
+		for axisType in [_chartConstants.xlCategory, _chartConstants.xlValue, _chartConstants.xlSeriesAxis]:
+			for axisGroup in [_chartConstants.xlPrimary, _chartConstants.xlSecondary]:
+				if self.wordChartObject.HasAxis(axisType, axisGroup):
+					self.elementKeyList.append(self._axisMap[axisType][axisGroup])
+					if self.wordChartObject.Axes(axisType, axisGroup).HasTitle:
+						self.elementKeyList.append(self._axisMap[axisType][axisGroup]+' Title')
+		self.elementKeyList.append('chartArea')
+		
+		self.elementKeyList.append('plotArea')
+
+		if self.wordChartObject.HasLegend:
+			self.elementKeyList.append('legend')
+
+		if self.wordChartObject.HasDataTable:
+			self.elementKeyList.append('dataTable')
+
+		self.chartElementsCount=len(self.elementKeyList)
+		super(WordChartElement,self).__init__(windowHandle=windowHandle, wordApplicationObject=wordApplicationObject, wordChartObject=wordChartObject, keyIndex=keyIndex)
+
+	def _get_name(self):
+		#Translators: Speak text chart elements when virtual row of chart elements is reached while navigation
+		return _("Chart Elements")
+
+	def _get_role(self):
+		return controlTypes.ROLE_CHARTELEMENT

```

Comment:
Changed it to a class variable.
Changed code is in branch in_t1541_Fin
Thanks a lot for the code review.